src/components/index.tsx

```diff
@@ -1,16 +1,24 @@
-import { createThemedComponent } from '../css/create-themed-component'
 import {
```

Comment:
Would you mind if we disregard this file for this PR, and instead merge from #15 once the issues mentioned there are resolved?

Reply:
Yes will do!